.gitignore

```diff
@@ -41,6 +42,8 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
+yarn.lock
+package-lock.json
```

Comment:
If we not tracking `package-lock.json` anymore, then it's best we remove that file from version control as well.

```
git rm package-lock.json
```

Reply:
This has been removed from version control by running the above command